tests/client.spec.ts

```diff
@@ -28,7 +28,7 @@ describe("client test", () => {
     };
 
     const myAction = {
-        actionType: ActionType.Command,
+        actionType: "COMMAND",
```

Comment:
declare the type of ICommandAction on the const than you won't need these many casts.

Reply:
Does this object literal fully implement ICommandAction?

Reply:
yes, the mandatory fields in ICommandAction are actionType and name